requirements.txt

```diff
@@ -1,10 +1,11 @@
 beautifulsoup4==4.13.3
+bs4==0.0.1
 dateparser==1.2.1
 emoji==2.14.1
-html2text==2024.2.26
+html_text==0.5.2
 puzpy==0.2.6
 requests==2.32.3
-Unidecode==1.3.8
+anyascii==0.3.1
 pyyaml==6.0.2
 xmltodict==0.14.2
-lxml==5.3.1
+lxml_html_clean>=0.4.1
```

Comment:
why is the lxml dependency removed here?

Reply:
Ah! I didn't know about the conditional dependency on lxml and thought it was only for the html_text. I probably got confused and thought I was getting html_text from lxml_html_clean instead. Reverted as you suggested.